cliquet/tests/test_cache.py

```diff
@@ -33,8 +33,8 @@ class BaseTestCache(object):
     backend = None
     settings = {}
 
-    def __init__(self, *args, **kwargs):
-        super(BaseTestCache, self).__init__(*args, **kwargs)
+    def setUp(self):
+        super(BaseTestCache, self).setUp()
```

Comment:
why would you change from init to setup ?

init : run once. setup: run before each test.